src/guillo-gmi/components/error_boundary.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+const style = { color: '#F44336', fontSize: 20, paddingBottom: 20 }
+
+export default class ErrorBoundary extends React.Component {
```

Comment:
Do you think makes sense to export this component on the library?

Reply:
For me doesn't make sense to export the ErrorBoundary directly to the lib because the people is going to use it always together with the Guillotina component... For me is part of the Guillotina component...  However, we can provide some way to use a custom ErrorBoundary instead of this default ErrorBoundary... What do you think about it?

Reply:
It's fine!